tests/operators/test_great_expectations.py

```diff
@@ -0,0 +1,149 @@
+"""
+Unit test module to test Great Expectations Operators.
+
+Requirements can be installed in, for instance, a virtual environment, with::
+
+    pip install -r requirements`
+
+and tests can be run with::
+
+    pytest -p no:warnings
+"""
+
+from contextlib import contextmanager
+import logging
+import os
+from pathlib import Path
+import unittest
+import unittest.mock as mock
+
+from airflow.exceptions import AirflowException
+from great_expectations_provider.operators.great_expectations import (
+    GreatExpectationsOperator,
+)
+from great_expectations.exceptions.exceptions import GreatExpectationsError
+
+
+log = logging.getLogger(__name__)
```

Comment:
Changing to be more consistent with the `great_expectations` codebase.
```suggestion
logger = logging.getLogger(__name__)
```

Reply:
See my other response around the use of `self.log` instead of the global logger instance...

Reply:
Gah, never mind.  Just noticed that this is the test.  LOL.  Will change from `log` to `logger`.